Library/Homebrew/dev-cmd/bottle.rb

```diff
@@ -462,7 +462,8 @@ def merge(args:)
       ohai formula_name
 
       bottle = BottleSpecification.new
-      bottle.root_url bottle_hash["bottle"]["root_url"]
+      root_url = args.root_url || bottle_hash["bottle"]["root_url"]
```

Comment:
```suggestion
      root_url = args.root_url.presence || bottle_hash["bottle"]["root_url"]
```

to be extra safe (e.g. on empty strings)